modules/andes-core/broker/src/main/java/org/wso2/andes/store/cassandra/HectorBasedAndesContextStoreImpl.java

```diff
@@ -672,6 +672,13 @@ public void close() {
     }
 
     /**
+     * {@inheritDoc}
+     */
+    @Override public void clearSlotStorage() throws AndesException {
+        throw new NotImplementedException();
+    }
+
```

Comment:
+1. Hector/Cassandra related classes will vanish very soon given that we are now fixed on RDBMS.